Assets/Scripts/BombBuddy.cs

```diff
@@ -3,6 +3,7 @@
 public class BombBuddy : MonoBehaviour
 {
     [SerializeField] private Sprite bombSprite;
+    public int bombID;
```

Comment:
I'm thinking we should default initialize this to something (0 or -1) so it doesn't break script behavior in new levels. @athanggupte thoughts?